README.md

```diff
@@ -246,32 +243,58 @@ using `cmake -D BUILD_SHARED_LIBS=OFF ..`.
   - **none**: This builds `sentry-native` without a backend, so it does not handle
     crashes. It is primarily used for tests.
 
-- `SENTRY_INTEGRATION_QT` (Default: OFF):
+- `SENTRY_INTEGRATION_QT` (Default: `OFF`):
   Builds the Qt integration, which turns Qt log messages into breadcrumbs.
 
-- `SENTRY_BREAKPAD_SYSTEM` (Default: OFF):
+- `SENTRY_BREAKPAD_SYSTEM` (Default: `OFF`):
   This instructs the build system to use system-installed breakpad libraries instead of the in-tree version.
 
-- `SENTRY_TRANSPORT_COMPRESSION` (Default: OFF):
+- `SENTRY_TRANSPORT_COMPRESSION` (Default: `OFF`):
   Adds Gzip transport compression. Requires `zlib`.
 
 - `SENTRY_FOLDER` (Default: not defined):
-  Sets the sentry-native projects folder name for generators that support project hierarchy (like Microsoft Visual Studio).
-  To use this feature, you need to enable hierarchy via [`USE_FOLDERS` property](https://cmake.org/cmake/help/latest/prop_gbl/USE_FOLDERS.html)
+  Sets the sentry-native projects folder name for generators that support project hierarchy (like Microsoft Visual
+  Studio). To use this feature, you need to enable hierarchy via [`USE_FOLDERS` property](https://cmake.org/cmake/help/latest/prop_gbl/USE_FOLDERS.html)
 
-- `CRASHPAD_ENABLE_STACKTRACE` (Default: OFF):
-  This enables client-side stackwalking when using the crashpad backend. Stack unwinding will happen on the client's machine
-  and the result will be submitted to Sentry attached to the generated minidump.
-  Note that this feature is still experimental.
+- `CRASHPAD_ENABLE_STACKTRACE` (Default: `OFF`):
+  This enables client-side stackwalking when using the crashpad backend. Stack unwinding will happen on the client's
+  machine and the result will be submitted to Sentry attached to the generated minidump. **Note that this feature is
+  still experimental**.
 
-- `SENTRY_SDK_NAME` (Default: sentry.native or sentry.native.android):
+- `SENTRY_SDK_NAME` (Default: `sentry.native` or `sentry.native.android`):
   Sets the SDK name that should be included in the reported events. If you're overriding this, also define
   the same value using `target_compile_definitions()` on your own targets that include `sentry.h`.
 
+- `SENTRY_HANDLER_STACK_SIZE` (Default: `64`):
+  This specifies the size in KiB of the stack reserved for the crash handler (including hooks like `on_crash` and
+  `before_send`) on Windows, Linux and the `inproc` backend in macOS. Reserving the stack is necessary in case of a
+  stack-overflow, where the handler could otherwise no longer execute. This parameter allows users to specify their
+  target stack size in KiB, because some applications might require a different value from our default. This
+  value can be as small as 16KiB (on `crashpad` and `breakpad`) for handlers to work, but we recommend 32KiB on 64-bit
```

Comment:
Why do we recommend 32KiB but set the default to 64? Is 32KiB the minimum recommendation?

Reply:
Yes, it is meant as a minimum recommendation. I'll make sure to explain that.